.rustfmt.toml

```diff
@@ -1,5 +1,4 @@
 edition = "2018"
 format_macro_matchers = true
 imports_granularity = "Crate"
-use_field_init_shorthand = true
-license_template_path = "./.license-template"
```

Comment:
why is this being removed?

Reply:
https://github.com/rust-lang/rustfmt/issues/5103